install/protected/upgrade_beta.sql

```diff
@@ -26,6 +26,16 @@ CREATE TABLE  `Currency` (
   UNIQUE KEY `currencyCode` (`currencyCode`)
 ) ENGINE=MyISAM DEFAULT CHARSET=latin1;
 
+DROP TABLE IF EXISTS `Fund`;

```

Comment:
Should the upgrade script drop this table? Shouldn't it only create the table if it doesn't already exist?